safehttp/plugins/xsrf/xsrfblockall/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package xsrfblockall provides a safehttp.Interceptor that ensures Cross-Site
```

Comment:
Package xsrfblockall provides a safehttp.Interceptor that protects against Cross-Site Request Forgery by rejecting all state-changing request.